src/Mod/PartDesign/Gui/TaskChamferParameters.cpp

```diff
@@ -328,6 +346,96 @@ void TaskChamferParameters::apply()
     }
 }
 
+void TaskChamferParameters::setupGizmos(ViewProviderDressUp* vp)
+{
+    if (!Gizmos::isEnabled()) {
+        return;
+    }
+
+    gizmos = std::make_unique<Gizmos>();
+
+    auto distanceGizmo = new Gui::LinearGizmo(ui->chamferSize);
+    auto twoDistanceGizmo = new Gui::LinearGizmo(ui->chamferSize);
+    auto angleGizmo = new Gui::RotationGizmo(ui->chamferAngle);
+
+    connect(ui->chamferType, qOverload<int>(&QComboBox::currentIndexChanged), [this, twoDistanceGizmo, angleGizmo] (int index) {
+        switch (index) {
+            case 0:

```

Comment:
don't we have some enum for that or a better way in general to handle that?

Reply:
Should be fixed.